src/test/java/org/zalando/nakadi/controller/EventTypeAuthorizationTest.java

```diff
@@ -0,0 +1,49 @@
+package org.zalando.nakadi.controller;
+
+import org.junit.Test;
+import org.zalando.nakadi.domain.EventType;
+import org.zalando.nakadi.exceptions.ForbiddenAccessException;
+import org.zalando.nakadi.utils.EventTypeTestBuilder;
+import org.zalando.problem.Problem;
+
+import javax.ws.rs.core.Response;
+import java.io.IOException;
+
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.doThrow;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+public class EventTypeAuthorizationTest extends EventTypeControllerTestCase {
+
+    public EventTypeAuthorizationTest() throws IOException {
+    }
+
+    @Test
+    public void whenPUTAuthorized200() throws Exception {
```

Comment:
It seems that this test is dependent on the implementation of the default authorization plugin (which always returns true). Should we perhaps mock the call to `isAuthorized`? Or am I over-complicating this?